app/Integrations/Packagist/Packages.php

```diff
@@ -171,7 +171,7 @@ public function download(array $package)
         $meta = stream_get_meta_data($temp);
         $path = $meta['uri'];
 
-        ($zip = new ZipArchive())->open($path);
+        with($zip = new ZipArchive())->open($path);
```

Comment:
Hmmm. Must be due to the variable assignment that php 5.5 doesn't support this. Normally it does support this syntax.